src/japi_pushsrv.c

```diff
@@ -158,11 +174,18 @@ void japi_pushsrv_subscribe(japi_context *ctx, int socket, json_object *jreq, js
 		return;
 	}
 	pushsrv_name = json_object_get_string(jval);
+	ret = json_object_object_get_ex(jreq,"socket",&jval);
+	socket = json_object_get_int(jval);
+	if (!ret | socket < 0) {
```

Comment:
Do you realy mean bitwise or (Just a single "|")? There is a complier warning. Or do you mean logic OR ("||")?

Reply:
Fixed, thanks